index.html

```diff
@@ -10,18 +10,13 @@
   </head>
 
   <body>
-    <header>
-      <nav class="header">
-        <ul class="nav-items">
-          <li class="logo"><a>Megha</a></li>
-          <li class="menu"><i class="fa-solid fa-bars"></i></li>
-        </ul>
-      </nav>
+    <header class="headercontainer">
+      <div class="logo">Megha</div>
+          <div><i class="fa-solid fa-bars hambrgricon"></i></div>
     </header>
```

Comment:
- [ ] Good job changing the `ul` to a `div`, one thing left, you should wrap the menu icon `div` with a `nav` tag instead, kindly change that. 😁